components/demo/demo-snippet.js

```diff
@@ -97,6 +120,16 @@ class DemoSnippet extends LitElement {
 	render() {
 		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
 		const skeleton = this._hasSkeleton ? html`<d2l-switch text="Skeleton" ?on="${this._skeletonOn}" @change="${this._handleSkeletonChange}"></d2l-switch>` : null;
+		const switches = html`
+			<d2l-switch text="RTL" ?on="${dirAttr === 'rtl'}" @change="${this._handleDirChange}"></d2l-switch><br />
+			<d2l-switch text="Fullscreen" ?on="${this._fullscreen}" @change="${this._handleFullscreenChange}"></d2l-switch><br />
+			${skeleton}`;
+		const settings = this.fullWidth && this._fullscreen ? html`
```

Comment:
Is this meant to be `this.fullWidth || this._fullscreen`?

Reply:
It's meant to be `&&` so that it only happens when you have opted for `fullWidth` and have enabled `_fullscreen` mode, but we could do it in `_fullscreen` regardless of `fullWidth`.

Reply:
I don't feel strongly either way. I've only used the fullscreen option a little, and so far it hasn't been an issue having the settings at the side. 🤷‍♂️